ESMF/GOCART2G_GridComp/DU2G_GridComp/DU2G_GridCompMod.F90

```diff
@@ -1194,8 +1187,7 @@ subroutine mie_(mie_table, nbins, nb, offset, q, rh, bext_s, bssa_s, basym_s, rc
      basym_s = 0.0d0
 
      do l = 1, nbins
-        call Chem_MieQuery(mie_table, l, real(offset+1.), q(:,:,:,l), rh, tau=bext, gasym=gasym, ssa=bssa)
-
+        call mie2g%Query(l, real(offset+1.), q(:,:,:,l), rh, tau=bext, gasym=gasym, ssa=bssa)
```

Comment:
```suggestion
        call mie2g%Query(l, offset+1., q(:,:,:,l), rh, tau=bext, gasym=gasym, ssa=bssa)
```
The `real()` in the expression is unnecesary.  Adding `1.` will force the conversion.